src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -90,6 +90,14 @@ export function updateLayeredMaterialNodeImagery(context, layer, node, parent) {
         }
     }
 
+    if (node.layerUpdateState[layer.id].isForcingUpdate()) {
+        // Create new MaterialLayer
+        nodeLayer = material.addLayer(layer);
+        // Init the new MaterialLayer by parent
+        const parentLayer = parent.material && parent.material.getLayer(layer.id);
+        nodeLayer.initFromParent(parentLayer, extentsDestination);
+    }
+
```

Comment:
Could you merge code with the `Initialisation`?
There is no forcing in elevation process?

Reply:
Added a `createMaterialLayer` function, and add the support for an `ElevationLayer`.